contracts/svm/programs/express_relay/src/lib.rs

```diff
@@ -546,3 +502,228 @@ pub struct Swap<'info> {
 
     pub relayer_signer: Signer<'info>,
 }
+
+#[derive(Accounts)]
+#[instruction(data: Box<SwapV2Args>)]
```

Comment:
this macro is only used to define some constraints, so the old way will magically work if the offset of fee_token is the same in `SwapV2Args` and `SwapArgs`